sdk/ai/ai-inference-rest/samples/cookbook/simpleCompletionsPage/src/index.js

```diff
@@ -8,7 +8,7 @@ function getInferenceChoices(resultDiv) {
 
   resultDiv.innerHTML = "";
   const credential = new AzureKeyCredential(azureKey.value);
-  const client = new ModelClient(endpoint.value, credential);
+  const client = ModelClient(endpoint.value, credential);
```

Comment:
Is there other samples where the same change is needed?

Reply:
Couldn't find any other matches for "new ModelClient"